pkg/cloud/aws/aws.go

```diff
@@ -56,25 +56,26 @@ type machineSetConfig struct {
 }
 
 type awsProvider struct {
-	workClinet    workclient.Interface
-	awsClinet     client.Interface
-	eventRecorder events.Recorder
-	region        string
-	infraId       string
-	ikePort       int64
-	nattPort      int64
-	clusterName   string
-	instanceType  string
-	gateways      int
-	cloudPrepare  cpapi.Cloud
+	workClinet        workclient.Interface
+	awsClinet         client.Interface
+	eventRecorder     events.Recorder
+	region            string
+	infraId           string
+	ikePort           int64
```

Comment:
ikePort can be removed as we are not longer using it. But it can be done in a separate PR.